cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1414,6 +1444,60 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     );
                 });
 
+                describe('Show correctness setting set as expected.', function() {
+                    var setShowCorrectness = function(show_correctness) {
+                        $('input[name=show-correctness][value=' + show_correctness + ']').click();
+                    }, expectWarningShown = function(shown) {
```

Comment:
How about a compromise?  3a8e81c fixes them to look like [existing code](https://github.com/open-craft/edx-platform/blob/881f867eb4ab0d6435fe5a366be213bdd5e8e17a/cms/static/js/spec/views/pages/course_outline_spec.js#L162), so as not to trip the linter.